package.json

```diff
@@ -11,18 +12,18 @@
   "author": "kiliman@gmail.com",
   "license": "ISC",
   "dependencies": {
-    "hash.js": "^1.1.3",
-    "npm": "^6.13.4",
-    "serialize-javascript": "^2.1.2",
+    "hash.js": "^1.1.7",
     "xml-formatter": "^1.0.1",
     "xmldom": "^0.1.27",
     "xpath": "0.0.24"
   },
   "devDependencies": {
-    "all-contributors-cli": "6.8.1",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-plugin-import": "^2.9.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.7.0"
+    "all-contributors-cli": "^6.19.0",
+    "eslint": "^7.15.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react-hooks": "^4.2.0"
```

Comment:
Might be nice to use a simpler ESLint config in the future since eslint-config-airbnb requires jsx-a11y, react, import, and react-hooks plugins.